### PR TITLE
Doc fixes: remove double placeholders; add them where missing.

### PR DESCRIPTION
**Pull request by ogorbacheva**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category:
- Doc fixes.

Tasks:
- https://st.yandex-team.ru/DOCAPI-3832

### CODE REVIEW

File: docs/en/query_language/functions/string_replace_functions.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ SELECT replaceRegexpAll('Hello, World!', '.', '\\0\\0') AS res
 
 ```
 ┌─res────────────────────────┐
-│ HHeelllloo,,  WWoorrlldd!! │
+│ HHeelllloo,, WWoorrlldd!! │
```
Comment (blinkov) [inline docs/en/query_language/functions/string_replace_functions.md]: you should have skipped this kind of ascii-tables, or maybe even all code blocks

**Review by blinkov — CHANGES_REQUESTED** [review]
—

File: docs/en/query_language/functions/string_replace_functions.md (hunk 1 above)
Comment (ogorbacheva) [reply docs/en/query_language/functions/string_replace_functions.md]: About all blocks not sure, cause could appear situation like this:
``` sql
SELECT Carrier, count(*) FROM ontime WHERE DepDelay>10**_ _**AND Year = 2007 GROUP BY Carrier ORDER BY count(*) DESC
```
Where '_' means space.
Should i ignore these spaces too?

Comment (blinkov) [reply docs/en/query_language/functions/string_replace_functions.md]: > Should i ignore these spaces too?
No, but to distinguish between these cases you'll probably have to review all of them manually.

File: docs/ru/operations/settings/settings.md [hunk 2]
```diff
@@ -16,10 +16,10 @@ ClickHouse применяет настройку в тех случаях, ко
 
 Возможные значения:
 
-- `deny`  - (по умолчанию) запрещает использование таких подзапросов (При попытке использование вернет исключение "Double-distributed IN/JOIN subqueries is denied");
```
Comment (ogorbacheva) [inline docs/ru/operations/settings/settings.md]: Here i will add spaces again in another commit to this PR.

File: docs/en/query_language/functions/string_replace_functions.md (hunk 1 above)
Comment (ogorbacheva) [reply docs/en/query_language/functions/string_replace_functions.md]: Ok

**Pull request merged**